fixieai/cli/validators.py

```diff
@@ -0,0 +1,17 @@
+import re
+
+AGENT_ID_PATTERN = re.compile(r"^\w+$")
+URL_PATTERN = re.compile(
+    r"^https?:\\/\\/(?:www\\.)?[-a-zA-Z0-9@:%._\\+~#=]{1,256}\\.[a-zA-Z0-9()]{1,6}\\b"
+    r"(?:[-a-zA-Z0-9()@:%_\\+.~#?&\\/=]*)$"
+)
+
+
+def validate_agent_id(agent_id: str):
```

Comment:
Technically this is an agent `handle`, not an agent ID. For consistency I think we should use the term `handle` here. The `agent_id` is in the format `username/handle`.